server.conf.in

```diff
@@ -109,6 +109,7 @@ serverRealm = local
 ; data: URL for images.
 ; The currently recommended CSP is:
 ;   default-src 'self';
+;   frame-src 'self' data:;
```

Comment:
For now this is required to load the sandbox iframe contents through the `data:` urls. Another possibility would be to use a separate FQDN and load the html part from there. However this would require additional configuration (besides having a second FQDN with a separate ssl cert).